package typed.protocol;

message smsg_error {
	required int32	error    = 1;	
}